btr-api/src/btr_api/resources/submission.py

```diff
@@ -222,22 +235,53 @@ def update_submission(sub_id: int):
+            except ExternalServiceException as err:
+                # Log error and continue to return successfully (does NOT block the submission)
+                current_app.logger.info(err.error)
+                current_app.logger.error('Error creating invoice for submission: %s', submission.id)
+
             try:
                 # NOTE: this will be moved out of this api once lear filings are linked
                 # update record in BOR (search)
                 token = btr_auth.get_bearer_token()
-                entity_addresses: dict[str, dict[str, dict]] = btr_entity.get_entity_info(
-                    jwt, f'{business_identifier}/addresses'
-                ).json()
-                entity['business']['addresses'] = [entity_addresses.get('registeredOffice', {}).get('deliveryAddress')]
+                address = btr_entity.get_entity_info(
+                    jwt, f'{business_identifier}/addresses?addressType=deliveryAddress').json()
+                entity['business']['addresses'] = [address['deliveryAddress']]
                 btr_bor.update_owners(submission, entity, token)
                 # update record in REG Search
                 btr_reg_search.update_business(submission, entity, token)
 
             except ExternalServiceException as err:
                 # Log error and continue to return successfully (does NOT block the submission)
                 current_app.logger.info(err.error)
-                current_app.logger.error('Error updating submission: %s', submission.id)
+                current_app.logger.error('Error updating search records for submission: %s', submission.id)
+
+            try:
+                # NOTE: will be moved to person or ownership table @event.listens_for after_insert #23291
+                # send emails out to newly added people
+                token = btr_auth.get_bearer_token()
+                delivery_address = btr_entity.get_entity_info(
+                    None, f'{business_identifier}/addresses?addressType=deliveryAddress', token).json()
+                business_contact = btr_auth.get_business_contact(business_identifier, token)
+                business_info = {**entity, **delivery_address, 'contact': {**business_contact}}
+
+                existing_people = [person['uuid'] for person in prev_submission_dict['payload']['personStatements']]
+                for person in submission.payload['personStatements']:
+                    # NOTE: unable to confirm information people may not have an email entered
+                    if person['uuid'] not in existing_people and person.get('email'):
+                        btr_email.send_added_to_btr_email(person, business_info, submission.effective_date, token)
+
+            except (BusinessException, ExternalServiceException) as err:
+                # Log error and continue to return successfully (does NOT block the submission)
+                current_app.logger.info(err.error)
```

Comment:
why log error message as info ?

Reply:
this is for debugging purposes. The line below is the error level message that gets sent to sentry for ops to look further into. Setting this to error level would spam sentry with multiple errors for a single issue